### PR TITLE
Bug 1877944: Show all resources for selected operator in the topology side panel

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
**Fixes**
https://issues.redhat.com/browse/ODC-4470

**Description**
Adds a managed by link and all the operator associated resources to the resources tab in the side panel in topology.

**Screenshots**
image

**Browser conformance**: 
- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

/kind bug

/cc @bgliwa01 @beaumorley @serenamarie125

### CODE REVIEW

**Review by bgliwa01 — APPROVED** [review]
lgtm

File: frontend/packages/operator-lifecycle-manager/src/components/operand/operand-link.tsx [hunk 1]
```diff
@@ -6,7 +6,8 @@ import { ClusterServiceVersionModel } from '../../models';
 
 export const OperandLink: React.SFC<OperandLinkProps> = (props) => {
   const { namespace, name } = props.obj.metadata;
-  const csvName = () =>
+  const csvName =
+    props.csvName ||
```
Comment (invincibleJai) [inline frontend/packages/operator-lifecycle-manager/src/components/operand/operand-link.tsx:10]: not part of this PR but would be good to use `csvNameFromWindow` here for `|| ` condition

File: frontend/packages/dev-console/src/components/topology/operators/TopologyOperatorBackedResources.tsx [hunk 2]
```diff
@@ -1,22 +1,153 @@
...
+  loaded,
+  loadError,
+  flatten,
+  linkForResource,
+}) => {
+  const manifestResources = flatten(resources);
+  return (
+    <StatusBox data={manifestResources} loaded={loaded} loadError={loadError} label="Resources">
+      <TopologyGroupResourcesPanel
+        manifestResources={flatten(resources)}
```
Comment (invincibleJai) [inline frontend/packages/dev-console/src/components/topology/operators/TopologyOperatorBackedResources.tsx:49]: in line 45 https://github.com/openshift/console/pull/6591/files#diff-fd65de433ab9e684dee076b3cc686df8R45, we have it it in const , can we use that?

File: frontend/packages/dev-console/src/components/topology/operators/TopologyOperatorBackedResources.tsx [hunk 3]
```diff
@@ -1,22 +1,153 @@
...
+  flatten: (resources: { [kind: string]: { data: K8sResourceKind[] } }) => K8sResourceKind[];
+  linkForResource?: (obj: K8sResourceKind) => React.ReactElement;
+};
+
+const OperatorResources: React.FC<OperatorResourcesProps> = ({
+  namespace,
+  resources,
+  loaded,
+  loadError,
+  flatten,
```
Comment (invincibleJai) [inline frontend/packages/dev-console/src/components/topology/operators/TopologyOperatorBackedResources.tsx:42]: just thought , why are we passing `flatten` as props across , can't we just use util `flattenCsvResources` wherever needed?

Comment (jeff-phillips-18) [reply frontend/packages/dev-console/src/components/topology/operators/TopologyOperatorBackedResources.tsx:42]: `flatten` is a function returned from a call to `flattenCsvResources` on line https://github.com/openshift/console/pull/6591/files#diff-fd65de433ab9e684dee076b3cc686df8R122

**Comment by invincibleJai** [comment]
@jeff-phillips-18 Thanks , looks good overall. In below scenario for `knative-eventing` i don't see `Job`

image

File: frontend/packages/dev-console/src/components/topology/operators/TopologyOperatorBackedResources.tsx (hunk 3 above)
Comment (invincibleJai) [reply frontend/packages/dev-console/src/components/topology/operators/TopologyOperatorBackedResources.tsx:42]: ah got it, my bad

**Comment by jeff-phillips-18** [comment]
@invincibleJai The side panel reuses code to show the same resources as instance's details page resources tab so I believe this PR to be correct. We should determine which is correct, topology or the details resource listing. I've added https://issues.redhat.com/browse/ODC-4886 to address that issue.
